sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ProtoDescriptorGenerator.scala

```diff
@@ -33,6 +34,7 @@ private[impl] object ProtoDescriptorGenerator {
     Array(
       AnyProto.getDescriptor,
       EmptyProto.getDescriptor,
+      Timestamp.getDescriptor.getFile,
```

Comment:
why not `TimestampProto.getDescriptor` ? Is there a difference actually?

Reply:
We need a `FileDescriptor` and `Timestamp.getDescriptor` returns `Descriptors.Descriptor`. I don't know the reason. Probably related with how the proto was defined?

Reply:
Yeah, but you can use `TimestampProto.getDescriptor` instead of `Timestamp.getDescriptor.getFile` I believe.. at least it worked when I was trying this a few weeks back